lib/rubocop/cop/internal_affairs/redundant_location_argument.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module InternalAffairs
+      # Checks for redundant `location` argument to `#add_offense`. `location`
+      # argument has a default value of `:expression` and this method will
+      # automatically use it.
+      #
+      # @example
+      #
+      #   # bad
+      #   add_offense(node, :expression)
+      #
+      #   # good
+      #   add_offense(node)
+      #   add_offense(node, :selector)
+      #   add_offense(node, :expression, MSG)
```

Comment:
Actually, this is not a `good` example. The default value for the 3rd argument to `#add_offense` is `nil`, which (in line 123 and 111-113) then default to `self.class::MSG`:
https://github.com/bbatsov/rubocop/blob/a7b8c1b5e8609aca83f4bdf4a2bcea5a9598f7d8/lib/rubocop/cop/cop.rb#L111-L123

I understand that you mean to exemplify that is is `good` to pass `:expression` as 2nd argument, as long as you pass a 3rd argument, too. You just need to change the 3rd argument to something other than `MSG` :-)

Reply:
@bquorning sure! Moreover, this example is listed as "bad" in the other cop, so it's a no go.

https://github.com/bbatsov/rubocop/blob/4abb67df0aa5b53cb42ceaae899636ef7cae2529/lib/rubocop/cop/internal_affairs/redundant_message_argument.rb#L12-L13